src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -19,30 +19,31 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
 
   @Input()
   model: ResponseTimeSeriesChartModel = {
-    durationMinutes: 60,
-    intervalMinutes: 1,
+    durationMinutes: 1440,
+    intervalSeconds: 10,
   };
 
-  interval: BehaviorSubject<number> = new BehaviorSubject<number>(1);
+  runInterval: BehaviorSubject<number> = new BehaviorSubject<number>(this.model.intervalSeconds);
 
   constructor(private feedbackResponseStatsService: FeedbackResponseStatsService) { }
 
   ngOnInit(): void {
-    this.interval.pipe(
-        switchMap((value: number) => interval(value * 60 * 1000)),
+    this.refresh();
+    this.runInterval.pipe(
+        switchMap((value: number) => interval(value * 1000)),
         tap(() => this.refresh()),
     ).subscribe();
   }
 
   refresh(): void {
-    this.feedbackResponseStatsService.loadResponseStats(this.model.durationMinutes.toString(),
-        this.model.intervalMinutes.toString())
-        .subscribe((data: FeedbackResponseRecords) => {
-          console.log(data)
-          const parsedData: any[] = this.parseData(data);
-          this.drawChart(parsedData, this.model.durationMinutes * 60 * 1000);
-        }, (e: ErrorMessageOutput) => {
-          console.log(e)
+    const durationSeconds: number = this.model.durationMinutes * 60;
+    this.feedbackResponseStatsService.loadResponseStats(durationSeconds.toString(),
+        this.model.intervalSeconds.toString())
+        .subscribe((records: FeedbackResponseRecords) => {
+          console.log(records);
+          this.drawChart(records.responseRecords, durationSeconds * 1000);
+        }, (err: ErrorMessageOutput) => {
+          console.log(err.error);
```

Comment:
console log messages can be removed afterwards, leave it there for now just for ease of testing

Reply:
Ok. General practice will be not to commit those lines and just keep it local.

Reply:
Do not attempt to try it with git cli. I heard it's a pain. Do it if your IDE (or sourceTree?) supports.